src/Radiance_Monitor/nwprod/gdas_radmon/driver/test_jgdas_verfrad_orion.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/ksh
+
+#SBATCH -o gdas_verfrad.o%j
+#SBATCH -J gdas_verfrad
+#SBATCH --ntasks=1 --mem=5g
+#SBATCH --time=20
+#SBATCH --account=da-cpu
+#SBATCH -D .
+
+
+set -x
+
+export MY_MACHINE=hera
```

Comment:
The script is for orion, while this variable says hera.

Reply:
Oh my.  And that it runs is a good indication it serves no purpose.  I'll remove it.